code/controllers/Processes/pipenet.dm

```diff
@@ -1,14 +1,16 @@
-var/global/list/datum/pipe_network/pipe_networks = list()
-var/global/list/obj/machinery/atmospherics/atmos_machines = list()
+var/list/datum/pipe_network/pipe_networks = list()
```

Comment:
In matters of purely style, I see no reason to make a fuss if PJB wants code it his way.